src/editor/core/plugins/link/index.js

```diff
@@ -41,7 +41,7 @@ const link = {
 
                 if (href[0] !== '#') {
                     const hrefUrl = new URL(href);
-                    if (hrefUrl.hostname !== window.location.hostname) {
+                    if (hrefUrl.hostname !== window.location.hostname || guid) {
```

Comment:
@yurabakhtin Does this mean, as soon as there is a "guid" parameter, the link will be opened as new window? Or are there any other conditions?

Reply:
@luke- If `guid` is provided then it is a file link(e.g. `[sample.pdf](file-guid:d1b583f8-fcc0-403c-97c9-862cbefab0ad)`).
I tested with this markdown code:

```
[sample.pdf](file-guid:d1b583f8-fcc0-403c-97c9-862cbefab0ad)

[another site](http://test.re/asd/asd "another site")

[this site](http://humhub.yb/sdf/sdf "this site")
```

It is rendered with new fix as:

```
<a href="http://humhub.yb/file/file/download?guid=d1b583f8-fcc0-403c-97c9-862cbefab0ad" data-file-guid="d1b583f8-fcc0-403c-97c9-862cbefab0ad" data-file-download="" data-file-url="http://humhub.yb/file/file/download?guid=d1b583f8-fcc0-403c-97c9-862cbefab0ad" target="_blank" rel="noopener noreferrer nofollow">sample.pdf</a>

<a href="http://test.re/asd/asd" title="another site" target="_blank" rel="noopener noreferrer nofollow">another site</a>

<a href="http://humhub.yb/sdf/sdf" title="this site" rel="noopener noreferrer nofollow">this site</a>
```

before this fix the first link had no `target="_blank"`.